src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -318,13 +323,20 @@
     default-so-linger-in-milliseconds))
 
 (schema/defn ^:always-validate
+  common-connector-config :- WebserverConnectorCommon
+  [config :- WebserverRawConfig]
+  {:request-header-max-size   (or (:request-header-max-size config)
+                                  default-request-header-size)
+   :so-linger-milliseconds    (so-linger-in-milliseconds config)
+   :idle-timeout-milliseconds (:idle-timeout-milliseconds config)})
```

Comment:
I understand that Jetty's default for idle timeout is currently 30 seconds.  Elsewhere in the tk-jetty9 service, though, I think we've more defensively coded in our own local default that we pass down to the Jetty configuration - rather than just relying upon default Jetty behavior.  I like this approach since it protects us from Jetty changing the default in a newer version without us noticing.  What would you think of coding the 30 second default in as a constant in tk-jetty9 and having tk-jetty9 always set a value for the connector's idle timeout?